app/features/feedback/shared/strategies/dud/create-rule.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'lodash';
+
+function createRule(subjectRule, workflowRule) {
+  const rule = {
+    failureEnabled: workflowRule.failureEnabled,
+    id: subjectRule.id,
+    strategy: workflowRule.strategy,
+    successEnabled: workflowRule.successEnabled
+  };
+
+  if (rule.failureEnabled) {
+    rule.failureMessage = subjectRule.failureMessage ||
+      workflowRule.defaultFailureMessage;
+  }
+
+  if (rule.successEnabled) {
+    rule.successMessage = subjectRule.successMessage ||
+      workflowRule.defaultSuccessMessage;
+  }
+
+  if (_.every(rule, property => property)) {
```

Comment:
I get the same error as noted in `radial/create-rule.js`, issue #2, with [the dud workflow (Feedback4)](https://feedback-update.pfe-preview.zooniverse.org/lab/1810/workflows/3136), after deleting the failure message

Reply:
Yeah - you should always have a message, so the error was in the validator. I've pushed a fix: dd089e6